infra/base-images/base-sanitizer-libs-builder/msan_build.py

```diff
@@ -396,19 +394,24 @@ def Build(self, package_name, output_directory, create_subdirs=False):
     extracted_paths = ExtractLibraries(deb_paths, self.work_dir,
                                        extract_directory)
     for extracted_path in extracted_paths:
-      if not os.path.islink(extracted_path):
-        PatchRpath(extracted_path, extract_directory)
+      if os.path.islink(extracted_path):
```

Comment:
This is the fix, rest is just yapf lint fix for passing presubmit.

Reply:
It doesn't seem to be working. With the latest images it seems `llvm-symbolizer` is still patched somewhere:
```sh
Running: docker run --rm --privileged -i -v /home/vagrant/oss-fuzz/build/out/selinux:/out -v /home/vagrant/oss-fuzz/build/work/selinux:/work -e FUZZING_ENGINE=libfuzzer -e SANITIZER=memory -e ARCHITECTURE=x86_64 -e FUZZING_LANGUAGE=c -e MSAN_LIBS_PATH=/work/msan gcr.io/oss-fuzz-base/base-sanitizer-libs-builder patch_build.py /out
...
Patching rpath for /out/llvm-symbolizer from $ORIGIN/../lib to $ORIGIN/instrumented_libraries/lib/x86_64-linux-gnu:$ORIGIN/../lib
```
```
$ sudo ./infra/helper.py check_build --sanitizer=memory selinux
Running: docker run --rm --privileged -i -e FUZZING_ENGINE=libfuzzer -e SANITIZER=memory -e ARCHITECTURE=x86_64 -e FUZZING_LANGUAGE=c -v /home/vagrant/oss-fuzz/build/out/selinux:/out -t gcr.io/oss-fuzz-base/base-runner test_all.py
INFO: performing bad build checks for /tmp/not-out/secilc-fuzzer
Broken fuzz targets 1
/tmp/not-out/secilc-fuzzer
BAD BUILD: /tmp/not-out/secilc-fuzzer seems to have either startup crash or exit:
/tmp/not-out/secilc-fuzzer -rss_limit_mb=2560 -timeout=25 -seed=1337 -runs=4 < /dev/null
INFO: Running with entropic power schedule (0xFF, 100).
INFO: Seed: 1337
INFO: Loaded 1 modules   (14853 inline 8-bit counters): 14853 [0x9c6248, 0x9c9c4d),
INFO: Loaded 1 PC tables (14853 PCs): 14853 [0x9c9c50,0xa03ca0),
INFO: -max_len is not provided; libFuzzer will not generate inputs larger than 4096 bytes
At least one initial sid must be defined in the policy
INFO: A corpus is not provided, starting from an empty corpus
At least one initial sid must be defined in the policy
#2	INITED cov: 285 ft: 286 corp: 1/1b exec/s: 0 rss: 67Mb
Symbol not inside parenthesis at line 1 of fuzz
	NEW_FUNC[1/1]: llvm-symbolizer: symbol lookup error: /tmp/not-out/instrumented_libraries/lib/x86_64-linux-gnu/libz.so.1: undefined symbol: __msan_va_arg_overflow_size_tls
==44==WARNING: external symbolizer didn't start up correctly!

ERROR: 100.0% of fuzz targets seem to be broken. See the list above for a detailed information.
Check build failed.
```